install/resources/settings.xml

```diff
@@ -409,15 +409,15 @@ max=9]]></optionscode>
 		</setting>
 		<setting name="uploadspath">
 			<title>Uploads Path</title>
-			<description><![CDATA[The path used for all board uploads. It <b>must be chmod 777</b> (on *nix servers).]]></description>
+			<description><![CDATA[The path used for all board uploads. It <b>must be chmod 777</b> (on *nix servers). Please note that unless the set path starting with a dot ".", the path will be converted to absolute. Default value: "./uploads"]]></description>
```

Comment:
```suggestion
			<description><![CDATA[The path used for all board uploads. It <b>must be chmod 777</b> (on *nix servers). If the path starts with a dot "." it will be converted to an absolute path. Default value: "./uploads" (relative path).]]></description>
```

Reply:
Just a minor nitpick: technically, the "relative path" condition is not "starts with a dot" but "does not start with a forward slash".